### PR TITLE
Create CODE_OF_CONDUCT.md

### PR DESCRIPTION
**Pull request by rymurr**
This change is [Reviewable](https://reviewable.io/reviews/projectnessie/nessie/875)

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+when an individual is representing the project or its community. Examples of
+representing a project or community include using an official project e-mail
+address, posting via an official social media account, or acting as an appointed
+representative at an online or offline event. Representation of a project may be
+further defined and clarified by project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting the project team at rymurr@gmail.com. All
```
Comment (rymurr) [inline CODE_OF_CONDUCT.md:58]: > rymurr@gmail.com

@laurentgo do we have a better email than this?

Comment (snazy) [reply CODE_OF_CONDUCT.md:58]: Maybe `private@projectnessie.org`?

Comment (rymurr) [reply CODE_OF_CONDUCT.md:58]: I thought private was a bit strong for this. Maybe community or pmc or something.

Comment (laurentgo) [reply CODE_OF_CONDUCT.md:58]: `codeofconduct@projectnessie.org` ?

Comment (snazy) [reply CODE_OF_CONDUCT.md:58]: conductcommittee@ ?

Comment (rymurr) [reply CODE_OF_CONDUCT.md]: i used `codeofconduct@projectnessie.org` can you create it Laurent?

**Review by laurentgo — APPROVED** [review]

**Pull request merged**